pkg/oracles/contracts/test/StableLPOracleMock.sol

```diff
@@ -27,8 +27,29 @@ contract StableLPOracleMock is StableLPOracle {
 
     function computeMarketPriceBalances(
         uint256 invariant,
-        int256[] memory prices
+        int256[] memory normalizedPrices
     ) public view returns (uint256[] memory) {
-        return _computeMarketPriceBalances(invariant, prices);
+        return _computeMarketPriceBalances(invariant, normalizedPrices);
+    }
+
+    function computeK(int256[] memory prices) public view returns (int256) {
+        (int256 a, int256 b) = _computeAAndBForPool(IStablePool(address(pool)));
+        return _computeK(a, b, prices);
+    }
+
+    function normalizePrices(int256[] memory prices) public view returns (int256[] memory normalizedPrices) {
+        int256 minPrice = prices[0];
+        for (uint256 i = 1; i < _totalTokens; i++) {
+            if (prices[i] < minPrice) {
+                minPrice = prices[i];
+            }
+        }
+
+        normalizedPrices = new int256[](_totalTokens);
+        for (uint256 i = 0; i < _totalTokens; i++) {
+            normalizedPrices[i] = _divDownInt(prices[i], minPrice);
```

Comment:
Here we do the same as in the actual contract but we don't fix the min price index to `FP1` :)